outlines/models/openai.py

```diff
@@ -313,81 +219,6 @@ async def call_api(prompt, system_prompt, config):
     return results, usage["prompt_tokens"], usage["completion_tokens"]
```

Comment:
We should consider removing the `@cache` decorator in

```
    @cache()
    async def call_api(prompt, system_prompt, config):
        responses = await client.chat.completions.create(
            messages=system_message + user_message,
            **asdict(config),  # type: ignore
        )
        return responses.model_dump()
```

Users might call `choice` multiple times to get multiple samples, and can manage their own cache if needed.